### ISSUE TITLE
Configuration to set MessageTemplate as the log record body

### ISSUE THREAD

**Issue by danelson** [serilog/serilog-sinks-opentelemetry#100]
Consider a log like the following:

```csharp
Log.LogInformation("This is a log with {data}", "This is a very long string that I don't want duplicated between an attribute and the log record body...")
```

The data property is duplicated between the log record body and the data attribute. This increases the cost of data transport and the cost of storage in the backend.

I would like an option where the "message template" is set to the log record body instead of the "rendered message" so that data is not duplicated. This would be like omitting the rendered message when using Serilog.Sinks.Console.

Naively I think that `IncludedData` could be updated to include an option such as `IncludedData.MessageTemplateAsLogBody`.

One partial workaround would be to modify the log record using the OpenTelemetry collector, but not everyone uses the collector and the initial data transport still happens.

**Comment by nblumhardt** [comment]
Thanks for the suggestion; definitely deserves some thought 👍

**Comment by danelson** [comment]
Well OpenTelemetry .NET just shipped 1.6.0 which makes the OTLP Log Exporter stable. It includes the option `IncludeFormattedMessage` ([link](https://github.com/open-telemetry/opentelemetry-dotnet/blob/92cdcc55f8e1bfe73f0ea3a6e841a6ad69cda0ca/src/OpenTelemetry/Logs/ILogger/OpenTelemetryLoggerOptions.cs#L44)).

**Comment by nblumhardt** [comment]
Merged in #104
